README.md

```diff
@@ -68,14 +69,15 @@ If there are changes made in the Original repository, you can sync the fork to k
 3. ```git checkout master``` to checkout your fork's local master branch.
 4. ```git merge upstream/master``` to sync your local master' branch with upstream/master. Note: Your local changes will be retained and your fork's master branch will be in sync with the upstream repository.
 
-#### Configure the app
+### Configure the app
 
 Maps app uses authenticated ArcGIS Online services for routing and geocoding. You need to configure the app for OAuth authentication.
 
-1. Log in to developers.arcgis.com.
+1. Log in to [developers.arcgis.com](https://developers.arcgis.com).
 2. Create a [new application](https://developers.arcgis.com/applications/new).
-3. After creating the application, navigate to the 'Authentication' tab and add a redirect URI. Note the Client ID and Redirect URI.
-4. In **Helpers\Configuration.cs** within the **MapsApp.Shared** project, replace the default client ID and Redirect URI with the values noted from step 3.
+3. After creating the application, navigate to the 'Authentication' tab and add a redirect URI. Any URI should work, but try to choose something unique to your app, like `my-maps-app://auth`.
```

Comment:
Is `my-maps-app://auth` the default redirect URI? If so, I would mention that it is... this way people can try out the app by adding the default redirect URI specified here.

Reply:
Default is `https://developers.arcgis.com` in code. I added a remark about that